src/QsFmt/Formatter/ParseTree/Namespace.fs

```diff
@@ -146,6 +187,29 @@ type NamespaceItemVisitor(tokens) =
 
     override _.VisitChildren node = Node.toUnknown tokens node |> Unknown
 
+    override _.VisitOpenElement context =
```

Comment:
It would be better if we didn't use these inner labels for the namespace elements. We should organize this NamespaceItemsVisitor similar to how the ParameterVisitor is structured for consistency.

Reply:
I have modified it. The repetitive `.directive` etc is gone, so the code has become clearer.

Reply:
Thanks that looks good now!